docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -401,6 +411,20 @@ Key Interfaces
 
         The EllipticCurvePublicKey object for this private key.
 
+    .. method:: sign(data, signature_algorithm)
+
+        .. versionadded:: 1.5
+
+        Sign one block of data which can be verified later by others using the
+        public key.
+
+        :param bytes data: The message string to sign.
+
+        :param signature_algorithm: An instance of a
+            :class:`EllipticCurveSignatureAlgorithm` provider.
+
+        :return: bytes: Signature.
```

Comment:
That's great, thanks.

On Sat, Jul 2, 2016 at 12:00 PM, Aviv Palivoda <notifications@github.com>
wrote:

> In docs/hazmat/primitives/asymmetric/ec.rst
> <https://github.com/pyca/cryptography/pull/3029#discussion_r69379603>:
>
> > @@ -401,6 +411,20 @@ Key Interfaces
> >
> >          The EllipticCurvePublicKey object for this private key.
> >
> > +    .. method:: sign(data, signature_algorithm)
> > +
> > +        .. versionadded:: 1.5
> > +
> > +        Sign one block of data which can be verified later by others using the
> > +        public key.
> > +
> > +        :param bytes data: The message string to sign.
> > +
> > +        :param signature_algorithm: An instance of a
> > +            :class:`EllipticCurveSignatureAlgorithm` provider.
> > +
> > +        :return: bytes: Signature.
>
> Fixed. I also changed this mistake at the DSA documentation. I hope this
> is OK that I did this in the same branch.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pyca/cryptography/pull/3029/files/2f973d491640c629ca2f6510dc40175134689486#r69379603>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAADBAxY55qj0sX_F44oyxCJxTDIYsQzks5qRosQgaJpZM4JCkqm>
> .
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6